views/lexicon/tabs/progress_bars.html.twig

```diff
@@ -216,5 +216,32 @@
         </a>
 
     </div>
+    <br/>
+    <h2 class="heading">Stepped Progress Bar</h2>
+
+    <div class="stepped-progressbar">
+        <div class="step"><span class="step-box current">1</span></div>
+        <div class="step"><span class="step-box">2</span></div>
+        <div class="step"><span class="step-box">3</span></div>
+        <div class="step last"><span class="step-box"><i class="icon-flag-checkered"></i></span></div>
+    </div><br/>
```

Comment:
Any reason why we don't use `margin` here rather than a `<br/>`

Reply:
This is just for the demo in Lexicon. Probably when it is applied it will have no br